README.md

```diff
@@ -89,19 +94,96 @@ int main()
 
   try
   {
-    auto client = BlobClient::CreateFromConnectionString(
-        GetConnectionString(), containerName, blobName);
+    // There are many types of client types, use the one that's appropriate
+    // for your scenario, such as BlobContainerClient.
+    BlobClient client
+        = BlobClient::CreateFromConnectionString(GetConnectionString(), containerName, blobName);
+    BlockBlobClient block = client.AsBlockBlobClient();
+
+    // Assuming there is some Azure::Core::IO::BodyStream containing the data to upload.
+    // For example either a MemoryBodyStream or FileBodyStream.
+    std::vector<uint8_t> data;
+    Azure::Core::IO::MemoryBodyStream stream(data);
+
+    Azure::Response<Models::UploadBlockBlobResult> response = block.Upload(stream);
+
+    Models::UploadBlockBlobResult model = response.Value;
+    printf("Last modified date of uploaded blob: %s\n", model.LastModified.ToString().c_str());
   }
   catch (const Azure::Core::RequestFailedException& e)
   {
+    printf("Status Code: %d, Reason Phrase: %s\n", (int)e.StatusCode, e.ReasonPhrase.c_str());

```

Comment:
and also, we do mix and match `std::cout` and `printf`, and also we call `printf()` not as `std:printf()` - I don't like it.
Btw, do our headers do `using namespace std` or `include <stdio.h>` instead of `include <cstdio>` so that our code sample can call `printf()` withot an `std::`? That should not be happening.

Reply:
Also, this statement uses a C-style cast. Should be `static_cast<int>(e.StatusCode)`.